docs/modules/ROOT/pages/meta-updater-testing.adoc

```diff
@@ -0,0 +1,44 @@
+= Testing
+
+//MC: No overlap with any content currently in the developer guide, but probably useful content to clean up and include.
```

Comment:
BWT this is mostly only interesting for contributors or people integrating meta-updater into another Yocto project.